src/GRANULAR/gran_sub_mod_tangential.cpp

```diff
@@ -123,16 +123,29 @@ void GranSubModTangentialLinearHistory::calculate_forces()
 
   // rotate and update displacements / force.
   // see e.g. eq. 17 of Luding, Gran. Matter 2008, v10,p235
+
+  // int newapproach = 0; //DRV: Original Method results in unphysical pendular motion
```

Comment:
For compile time conditional code, please use C style preprocessor directives.

Reply:
This is temporary and will be removed in the next version, so I have not modified it for now.